unlock-protocol.com/src/components/content/HomeContent.js

```diff
@@ -49,98 +53,120 @@ export const HomeContent = () => {
         <OpenGraphTags />
       </Head>
 
-      <Hero>
-        <NoPhone>
-          <img alt="Unlock" src="/static/images/pages/svg/hero.svg" />
-        </NoPhone>
-        <Actions>
-          <h1>Monetize the Web with memberships</h1>
-          <Buttons>
-            <HomepageButton
-              label="Dashboard"
-              destination={`${
-                getConfig().publicRuntimeConfig.unlockApp
-              }/dashboard`}
+      <Grid>
+        <Hero>Monetize the web through Memberships</Hero>
```

Comment:
Should we make the casing more consistent on this? Either title case, or by downcasing “Memberships”